scripts/functions_test.py

```diff
@@ -209,20 +206,18 @@ def main(argv):
 
         # call CF
         logger.debug('Sending input json: %s', json.dumps(testInputJson, ensure_ascii=False).encode('utf8'))
-        response = requests.post(
-            testUrl, 
-            auth=(username, password), 
-            headers={'Content-Type': 'application/json'}, 
-            data=json.dumps(testInputJson, ensure_ascii=False).encode('utf8'))
-
-        responseContentType = response.headers.get('content-type')
-        if responseContentType != 'application/json':
-            logger.error('Response content type is not json, content type: %s, response:\n%s', responseContentType, response.text)
-            continue
+
+        testOutputReturnedJson = getFunctionResponseJson(url + '/' + namespace,
+                                               username,
+                                               password,
+                                               (test['cf_package'] if 'cf_package' in test else package),
+                                               (test['cf_function'] if 'cf_function' in test else function),
+                                               '?blocking=true&result=true',
```

Comment:
I do not like so much that by 'parameters' we actually think query string even with '?' at the beginning... I would rename it to query string or expect really parameters -> dict... and remove the question mark.

Reply:
Changed to dict.